activate.sh

```diff
@@ -34,6 +34,17 @@ AUTOENV_ENV_LEAVE_FILENAME="${AUTOENV_ENV_LEAVE_FILENAME:-.env.leave}"
 AUTOENV_VIEWER="${AUTOENV_VIEWER:-cat}"
 # AUTOENV_ENABLE_LEAVE
 
+# @internal
+__autoenv_cd() {
+	if [ "${__autoenv_has_builtin}" = 'yes' ]; then
+		\builtin cd "${1}" || return
```

Comment:
This is always forwarding one arg to the built-in. Shouldn't this be `"$@"`? Wouldn't that fix #254?